### PR TITLE
Dump db on issued command

### PR DESCRIPTION
**Pull request by proof3**
Screenshot 2023-02-27 at 5 27 07 PM

Here is an example of what the issued commands output currently looks like. it just prints out all the rows of the database.
I can change it so that data for each user is shown on one column.

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -14,13 +14,17 @@ export const SELECT_LAST_WEEK =
...
+    "SELECT * FROM userweights";
+
 
 // Command Labels
 export const HELP_CMD    = "help";
 export const INSPIRE_CMD = "inspire";
 export const DEPRESS_CMD = "depress";
 export const RECORD_WEIGHT_CMD = "recordWeight";
 export const STATS_CMD = "stats";
+export const ISSUED_CMD = "issued";
```
Comment (Merctron) [inline src/constants.js]: Why is the command called `issued`?

File: src/constants.js [hunk 2]
```diff
@@ -14,13 +14,17 @@ export const SELECT_LAST_WEEK =
     "SELECT * FROM userweights WHERE username LIKE ? AND userweights.date " +
     "BETWEEN strftime('%Y%m%d',date('now', '-14 days')) AND " +
     "strftime('%Y%m%d',date('now', '-7 days')) ORDER BY date DESC LIMIT 7";
+export const SELECT_TABLE =
```
Comment (Merctron) [inline src/constants.js]: Can we delete this?

File: src/db-utils.js [hunk 3]
```diff
@@ -64,4 +66,9 @@ export async function weightStats(username) {
             res(data);
         });
     });
+}
+
+export function dumpDB() {
+    const data = fs.readFileSync(DB_LOCATION, {encoding:'utf8', flag:'r'});
```
Comment (Merctron) [inline src/db-utils.js]: We don't need to block the PR on this but we need to enforce a maximum for the file size we are willing to dump.

File: src/constants.js (hunk 1 above)
Comment (proof3) [reply src/constants.js]: You told me to call it issued on discord? I think I might have misunderstood what you were saying.

Comment (proof3) [reply src/constants.js]: I have changed the command to dumpDB

File: tst/command-processor.test.js [hunk 4]
```diff
@@ -1,9 +1,52 @@
...
+      }
+    }
+  ));
+
+});
+
+test("Process dumpDB command on file grater than 8MB", async () => {
+
+  const data = fs.readFileSync(DB_LOCATION, {encoding:'utf8', flag:'r'});
+  var newData = "";
```
Comment (Merctron) [inline tst/command-processor.test.js]: We should prefer `let` over `var`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let#scoping_rules

**Comment by Merctron** [comment]
We are seeing test failures: https://github.com/Merctron/WhaleBot/actions/runs/4341452022

**Pull request merged**